quickstart/csharp/uwp/keyword-recognizer/README.md

```diff
@@ -0,0 +1,35 @@
+# Quickstart: Recognize keywords in C# under Universal Windows Platform
+
+This sample demonstrates how to recognize keywords with C# under the Universal Windows Platform using the Speech SDK.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-csharp-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+## Prerequisites
+
+* A Windows PC with Windows 10 Fall Creators Update (10.0; Build 16299) or later and with a working microphone.
+* [Microsoft Visual Studio 2017](https://www.visualstudio.com/), Community Edition or higher.
+* The **Universal Windows Platform development** workload in Visual Studio.
+* Note: processor target ARM or ARM64 is not yet supported.
+
+  You can enable it in **Tools** \> **Get Tools and Features**.
+
+## Build the sample
+
+* **By building this sample you will download the Microsoft Cognitive Services Speech SDK. By downloading you acknowledge its license, see [Speech SDK license agreement](https://aka.ms/csspeech/license201809).**
+* [Download the sample code to your development PC.](/README.md#get-the-samples)
+* Start Microsoft Visual Studio 2017 and select **File** \> **Open** \> **Project/Solution**.
+* Navigate to the folder containing this sample, and select the solution file contained within it.
+* Set the active solution configuration and platform to the desired values under **Build** \> **Configuration Manager**:
+  * On a 64-bit Windows installation, choose `x64` as active solution platform.
+  * On a 32-bit Windows installation, choose `x86` as active solution platform.
+* Press Ctrl+Shift+B, or select **Build** \> **Build Solution**.
+
+## Run the sample
+
+To debug the app and then run it, press F5 or use **Debug** \> **Start Debugging**. To run the app without debugging, press Ctrl+F5 or use **Debug** \> **Start Without Debugging**.
+
+When you see the `Enable microphone` button, click it. Then select the `Listen for Keyword`. At this point you can say the keyword **"Computer"**. Once the keyword is detected, you can start talking and once you are done you can click the button `Save to File` to save the audio that was just recorded to a file.
```

Comment:
> Then select the `Listen for Keyword`. [](start = 54, length = 38)

is there a selection for "Listen for Keyword" ?

Reply:
It is a button